helpers/auth/token_authentication.py

```diff
@@ -3,16 +3,14 @@
 
 
 class Authentication:
```

Comment:
When a module only has one class, it is advisable to give it a name that resembles the name of the main class in it